esp-hal/src/spi/master.rs

```diff
@@ -100,6 +96,18 @@ pub enum SpiInterrupt {
     /// This interrupt is triggered when an SPI transaction has finished
     /// transmitting and receiving data.
     TransDone,
```

Comment:
```suggestion
    TransmitDone,
```

I couldn't find anything in the Rust guidelines here, but I'm not sure how much we're saving by cutting off three chars. I think we should use the full word here (and in general avoid abbreviations in the API)

Reply:
It's Transfer, though :)

Reply:
sounds good - what about `DmaSegTransDone`? Should it be `DmaSegmentedTransferDone`?

I _guess_ `App1` and `App2` are not too bad

Reply:
I think `App` has embedded itself into the language by this time.

Reply:
> Transfer

Oops, all the more reason to spell it fully out :D.

> Should it be DmaSegmentedTransferDone

Yes, I'll make a PR with some guideline updates here.

> I guess App1 and App2 are not too bad

Yep, they're fine